src/sentry/static/sentry/app/views/organizationEvents/utils/eventsRequest.jsx

```diff
@@ -108,6 +113,7 @@ class EventsRequest extends React.PureComponent {
     limit: 15,
     getCategory: i => i,
     query: '',
+    groupId: '',
```

Comment:
Empty string for group ID seems like an odd default value since it's not actually a valid group ID, would `null` or empty list be better?